modules/imgproc/src/imgwarp.cpp

```diff
@@ -5410,63 +5431,10 @@ class WarpAffineInvoker :
     int interpolation, borderType;
     Scalar borderValue;
     int *adelta, *bdelta;
-    double *M;
+    const double *M;
 };
 
 
-#if defined (HAVE_IPP) && IPP_VERSION_X100 >= 810 && IPP_DISABLE_BLOCK
```

Comment:
@mshabunin, we do not want to be accused by Intel that we remove IPP from OpenCV. We promised no to do so. Can you please, retain the code, comment it off and supply explaining comment that the API is not available anymore and thus this code needs to be updated? The same should be done for warpPerspective and other places where IPP was used and now is not